src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,143 @@
+.card-conteiner {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.img-box__img {
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.img-box {
+  height: 166px;
+}
+
+.name {
+  color: $dark-grey;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card-conteiner__name {
+  margin-top: 40px;
+  margin-left: 16px;
+}
+
+.code {
+  color: #616070;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card-conteiner__code {
+  margin-top: 4px;
+  margin-left: 16px;
+}
+
+.review {
+  display: flex;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card-conteiner__review {
+  margin-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  &-lable {
+    color: #616070;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  &-num {
+    color: $dark-grey;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 18px;
+  }
+}
+
+.card-conteiner__price {
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 24px;
+}
+
+.bay-link-box {
+  align-self: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  background: $blue;
+  border: 1px solid $blue;
+  border-radius: 5px;
+}
+
+.card-conteiner__bay-link-box {
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+.link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.bay-link-box:hover .link {
+  color: $blue;
+}
+
+.bay-link-box:hover {
```

Comment:
Check the 9th point in the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)